Airports/EGCC/SMR/Geo.txt

```diff
@@ -98,185 +98,191 @@ N053.22.04.850 W002.16.59.239 N053.22.03.403 W002.17.00.289 smrCCstand
-; - 47
+;47
 N053.21.32.637 W002.16.13.776 N053.21.30.995 W002.16.11.114 smrCCstand
-; - 48
+;48
 N053.21.33.255 W002.16.10.117 N053.21.33.428 W002.16.12.853 smrCCstand
-; - 49
+;49
 N053.21.34.752 W002.16.09.637 N053.21.35.429 W002.16.12.589 smrCCstand
-; - 50
+;50
 N053.21.35.846 W002.16.08.462 N053.21.36.821 W002.16.10.145 smrCCstand
-; - 51
+;51
 N053.21.36.816 W002.16.06.999 N053.21.37.782 W002.16.08.663 smrCCstand
-; - 52
+;52
 N053.21.37.633 W002.16.05.290 N053.21.38.702 W002.16.07.138 smrCCstand
-; - 53
+;53
 N053.21.39.684 W002.16.05.562 N053.21.38.498 W002.16.03.520 smrCCstand
-; - 54
+;54
 N053.21.39.443 W002.16.01.588 N053.21.40.790 W002.16.03.758 smrCCstand
-; - 55
+;55
 N053.21.40.544 W002.15.59.285 N053.21.42.195 W002.16.01.141 smrCCstand
-; - 55L
+;55L
 N053.21.41.415 W002.16.01.214 N053.21.40.347 W002.16.00.078 smrCCstand
-; - 55R
+;55R
 N053.21.42.112 W002.16.00.023 N053.21.40.788 W002.15.58.685 smrCCstand
-; - 57
+;57
 N053.21.42.991 W002.15.55.366 N053.21.41.911 W002.15.53.923 smrCCstand
-; - 58
+;58
 N053.21.42.807 W002.15.52.065 N053.21.43.882 W002.15.53.520 smrCCstand
-; - 100
+;100
 N053.21.36.113 W002.16.39.124 N053.21.35.987 W002.16.36.897 smrCCstand
-; - 101
+;101
 N053.21.33.437 W002.16.39.570 N053.21.32.263 W002.16.37.523 smrCCstand
-; - 61
+;61
 N053.21.33.343 W002.16.45.065 N053.21.33.602 W002.16.49.209 smrCCstand
-; - 62
+;62
 N053.21.37.263 W002.16.44.742 N053.21.36.712 W002.16.48.552 smrCCstand
-; - 63
+;63
 N053.21.39.489 W002.16.47.694 N053.21.37.900 W002.16.50.332 smrCCstand
-; - 64
+;64
 N053.21.41.202 W002.16.50.565 N053.21.39.758 W002.16.52.946 smrCCstand
-; - 65
+;65
 N053.21.42.856 W002.16.53.435 N053.21.41.092 W002.16.56.368 smrCCstand
-; - 66
+;66
 N053.21.44.845 W002.16.57.127 N053.21.43.329 W002.16.59.657 smrCCstand
-; - 67
+;67
 N053.21.46.714 W002.17.00.295 N053.21.45.067 W002.17.03.017 smrCCstand
-; - 68
+;68
 N053.21.48.590 W002.17.03.432 N053.21.47.009 W002.17.06.041 smrCCstand
-; - 69
+;69
 N053.21.50.466 W002.17.06.585 N053.21.49.017 W002.17.08.966 smrCCstand
-; - 70
+;70
 N053.21.52.321 W002.17.09.729 N053.21.50.879 W002.17.12.118 smrCCstand
-; - 71
+;71
 N053.21.54.194 W002.17.12.867 N053.21.52.533 W002.17.15.597 smrCCstand
-; - 72
+;72
 N053.21.56.082 W002.17.17.104 N053.21.54.509 W002.17.19.710 smrCCstand
-; - 73
+;73
 N053.21.58.029 W002.17.20.333 N053.21.56.402 W002.17.23.084 smrCCstand
-; - 74 (WFU AIRAC 1708)
-;N053.21.59.955 W002.17.23.568 N053.21.58.307 W002.17.26.319 smrCCstand
-; - 86
+;74 
+N053.21.59.955 W002.17.23.568 N053.21.58.307 W002.17.26.319 smrCCstand
+;86
 N053.22.02.602 W002.17.15.898 N053.22.00.947 W002.17.18.648 smrCCstand
-; - 85
+;85
 N053.22.00.687 W002.17.12.641 N053.21.59.032 W002.17.15.411 smrCCstand
-; - 84
+;84
 N053.21.55.619 W002.17.08.085 N053.21.57.036 W002.17.05.741 smrCCstand
-; - 83
+;83
 N053.21.55.171 W002.17.02.571 N053.21.53.748 W002.17.04.947 smrCCstand
-; - 82
+;901
+N053.22.02.437 W002.17.24.699 N053.22.04.097 W002.17.27.708 smrCCstand
+;903
```

Comment:
This stand comment is fine though assuming all three of these lines relate to 903